CMakeLists.txt

```diff
@@ -216,9 +226,24 @@ endif()
 if (UNIX)
   add_subdirectory(docs/refman)
   add_subdirectory(pkgconfig)
-  add_subdirectory(samples)
 endif()
 
+if (BUILD_ENCLAVES)
+  add_subdirectory(samples)
+endif ()
+
+if (WIN32)
+  install(FILES ./scripts/clangw ./scripts/llvm-arw
+    DESTINATION ${CMAKE_INSTALL_BINDIR}/scripts/)
+  install(FILES ./scripts/install-windows-prereqs.ps1
```

Comment:
Yes, we should track that we need to sign this. Thanks Simon!

Radhika and I talked about this: we're going to publish an install_oe_sdk-windows.md file that describes how to configure your system, but also gives you the option of running this script with another option to configure your system for SDK consumption. That's because a user will be expected to install 7z, nuget, Visual Studio, OpenSSL, LLVM/clang, the Intel PSW executable, the Intel DCAP executable (and all of its internals, like the nuget packages), and a DCAP client.

I imagine it may be difficult to expect a user to install all of this manually, so I think providing a script to do this will make a user's experience easier. We'll probably just offer another option to this script (probably rename LaunchConfiguration -> Configuration) because the above dependencies are just a subset of the contributor dependencies on Windows.

Reply:
Yes, we will add install_oe_sdk-windows.md.